.github/actions/install_binstall/action.yaml

```diff
@@ -0,0 +1,12 @@
+name: 'Installs binstall'
+description: |
+  Installs `cargo binstall` at the version specified in dfx.json
```

Comment:
The version is specified in `versions.bash`, not dfx.json

Reply:
Thank you.  Good catch.  Done.